biz.aQute.bnd/src/aQute/bnd/main/RemoteCommand.java

```diff
@@ -109,6 +111,57 @@ public void event(Event e) throws Exception {
 		agent = launcher.getAgent();
 	}
 
+	@Description("List the bundles already installed in remote framework")
+	@Arguments(arg = {})
+	interface ListOptions extends Options {
```

Comment:
Could you add an option to get output in JSON?

Reply:
Done

Reply:
Done